sdcm/logcollector.py

```diff
@@ -958,6 +958,8 @@ class SCTLogCollector(LogCollector):
                 search_locally=True),
         FileLog(name='argus.log',
                 search_locally=True),
+        FileLog(name=r'*debug.json',
```

Comment:
would better if it's more specific about the `data_validator` files.

Reply:
the files name will start with view name and ended with "debug". So I search it with`*debug` as common part of the file name